src/zino/scheduler.py

```diff
@@ -74,9 +79,9 @@ def init_state_for_devices(devices: Sequence[PollDevice]):
 
 
 async def load_and_schedule_polldevs(polldevs_conf: str):
-    new_devices, deleted_devices = load_polldevs(polldevs_conf)
-    schedule_new_devices(new_devices)
-    deschedule_deleted_devices(deleted_devices)
+    new_devices, deleted_devices, changed_devices = load_polldevs(polldevs_conf)
+    deschedule_deleted_devices(deleted_devices | changed_devices)
+    schedule_new_devices(new_devices | changed_devices)
```

Comment:
`schedule_new_devices` will make a log message saying `_log.debug("Scheduling %s new devices", len(devices))`, which will be a bit misleading when some of the "new devices" are existing devices that are being re-scheduled in order to update them. Could these two be separated somehow, that its clear that some are new devices and some are existing ones that are updated?

Reply:
Would it be enough in your opinion to change the log message to `Scheduling %s new/changed devices`? Because in `load_polldevs` we are logging which devices are new and which are changed (`_log.info("loaded new devices: %r", new_devices)`), so we will have that information just above in the logs.

Reply:
Or I could write a separate function that does actually change the existing jobs instead of deleting the existing jobs and creating new ones.

Reply:
`new/changed` is probably fine in that case